src/components/client/campaigns/CampaignDetails.tsx

```diff
@@ -121,6 +138,11 @@ export default function CampaignDetails({ campaign }: Props) {
         <Grid item xs={12}>
           <Divider />
         </Grid>
+        <Grid container item xs={12}>
+          {Object.entries(chipsLabel).map(([id, label]) => (
+            <CampaignDetailsChip key={id} chip={label} onClick={() => scrollToSection(id)} />
+          ))}
+        </Grid>
```

Comment:
Looks good! Just let's move them directly below the title and also let's use color=primary ot make the main blue color of the site

Reply:
I contact the ui/ux college to clarify translations and design(colors etc..)

Reply:
![Screenshot 2023-08-05 182515](https://github.com/podkrepi-bg/frontend/assets/68897146/5e36276a-55d7-4ce4-b42b-3c55a6b8bfe4)
![Screenshot 2023-08-05 182453](https://github.com/podkrepi-bg/frontend/assets/68897146/79480091-e1fb-43d5-995e-66b504f94a65)
i think is a good starting point(with some bugs for fixing)